### ISSUE TITLE
Timezone tokens Z and ZZ works wrong

### ISSUE THREAD

**Issue by antonovicha** [taylorhakes/fecha#80]
Token 'Z' produces 'Z' as an output (i.e. it is not recognized).

Token 'ZZ" produces time offset without `:`:
```javascript
console.log(fecha.format(new Date("2017-03-11T02:30:00.000+07:00"), "ZZ"));
// +0100
```
It also uses current machine timezone. Because `new Date().getTimezoneOffset();` always returns current machine timezone.

fecha version 4.1.0 .

**Comment by taylorhakes** [comment]
Hi @antonovicha. Thank you for the bug report. I will take a look at the Z token tonight.

The ZZ token is not suppose to not have a colon. It is documented in the readme. It will also always show the local timezone because all dates are represented in local time by default, since there is no way to represent the timezone in the Javascript date object. You can always string concat any timezone you want if local doesn’t work for you.

**Comment by taylorhakes** [comment]
This should be fixed in 4.2.0. Thanks for the report
